consensus/polybft/consensus_runtime.go

```diff
@@ -39,6 +39,18 @@ type Transport interface {
 	Gossip(message interface{})
 }
 
+// txPoolInterface is an abstraction of transaction pool
+type TxPoolInterface interface {
+	Prepare()
+	Length() uint64
+	Peek() *types.Transaction
+	Pop(tx *types.Transaction)
+	Drop(tx *types.Transaction)
+	Demote(tx *types.Transaction)
+	ResetWithHeaders(headers ...*types.Header)
+	SetSealing(bool)
```

Comment:
Seems like we are not using these functions in the polybft, so maybe we should remove those from this interface?